packages/react-reconciler/src/__tests__/ReactSuspenseList-test.internal.js

```diff
@@ -2135,4 +2135,159 @@ describe('ReactSuspenseList', () => {
       </div>,
     );
   });
+
+  it('eventually resolves two nested forwards suspense list with a hidden tail', async () => {
```

Comment:
nit: "forwards suspense list**s**"

Reply:
oops. this is a copy paste of the test above.